src/StackAdmin/Compute/Commands.Compute/Commands.Compute.csproj

```diff
@@ -338,9 +338,7 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <None Include="..\AzureRM.Compute.psd1">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
@bganapa do we not want to copy this file to the output directory?

Reply:
good catch!.. added copy always